svelte/sveltekit-starter/dfx.json

```diff
@@ -19,7 +19,7 @@
 			"packtool": ""
 		}
 	},
-	"dfx": "0.12.0",
+	"dfx": "0.14.3",
```

Comment:
```suggestion
```
I'd say removing that line is even better so we don't have to update the version all the time